### PR TITLE
Tracing fish dabs

### PR DESCRIPTION
**Pull request by TimDaub**
Plausible.io allows to publicly share the stats. It's fery good.

### CODE REVIEW

**Comment by TimDaub** [comment]
@vrde do you think we should create a DIP first?

**Comment by TimDaub** [comment]
@vrde You might wanna look at this page: https://plausible.io/data-policy

**Comment by TimDaub** [comment]
bump

File: _config.yml [hunk 1]
```diff
@@ -102,7 +102,7 @@ ignore:
 # Extensions
 ## Plugins: https://hexo.io/plugins/
 ## Themes: https://hexo.io/themes/
-theme: geek
+theme: Nlvi
```
Comment (vrde) [inline _config.yml:105]: Is this part of the PR?

Comment (TimDaub) [reply _config.yml:105]: I think that's a mistake I should fix

Comment (vrde) [reply _config.yml:105]: BTW I'd be happy to try another theme, so if you want to move it to another PR I'd be good with that.

Comment (TimDaub) [reply _config.yml:105]: Yes, the responsiveness of the geek theme is quite terrible.

I'll create an issue

**Comment by TimDaub** [comment]
> I see that in the PR there is also a change in the theme, is this part of it?

Fixed.

**Pull request merged**